oracle/src/oracleimporter.cpp

```diff
@@ -365,7 +365,7 @@ int OracleImporter::importCardsFromSet(const CardSetPtr &currentSet,
 
             // add other face for split cards as card relation
             if (!getStringPropertyFromMap(card, "side").isEmpty()) {
-                properties["cmc"] = getStringPropertyFromMap(card, "faceConvertedManaCost");
+                properties["cmc"] = getStringPropertyFromMap(card, "manaValue");
```

Comment:
This isn't correct. `faceManaValue` should be used, as this is a face product

Reply:
https://mtgjson.com/data-models/card/card-set/#facemanavalue

Reply:
I'm totally aware that faceConvertedManaCost is deprecated and superseded by faceManaValue. See the PR description and discussions in the linked issue (as well as the mentioned MTGJSON issue).

As noted, `When testing with faceManaValue it did not solve the issue though.`
Which is no surprise as it hold the same data as faceConvertedManaCost.

I'm not saying this is the right solution here, and this needs a check by somebody more familiar with the code and logic around this.
But the result it gives, appears to be better and I did not spot issues yet.

Reply:
Going step by step now, ensuring we save the right data first.

But if MTGJSON has all data and fields correct (which still needs to be verified), then Cockatrice is probably displaying certain data not correctly.

Reply:
MTGJSON is inconsistent (not even manaValue is present on all cards like the first instances of Blex, or my parser needs work) and Cockatrice should use the back face MV for the back face. https://github.com/Cockatrice/Cockatrice/issues/4271#issuecomment-1849757496

Reply:
Let's find all data issues related to the problems in Cockatrice and report them at MTGJSON.

Then we can see how to fix stuff in Cockatrice.